lbrynet/lbrynet_daemon/LBRYDaemonCLI.py

```diff
@@ -21,8 +34,28 @@ def main():
         print "lbrynet-daemon isn't running"
         sys.exit(1)
 
-    args = sys.argv[1:]
-    meth = args[0]
+    parser = argparse.ArgumentParser()
+    parser.add_argument('method', nargs=1, type=str)
+    parser.add_argument('params', nargs=argparse.REMAINDER, default=None)
+    args = parser.parse_args()
+    meth = args.method[0]
+    params = {}
+    if args.params:
+        if len(args.params) > 1:
```

Comment:
This code would be nicer as its own function - which could/should then be unit-tested